src/pages/TeamPage/TeamPage.js

```diff
@@ -1,14 +1,90 @@
-import React, { useContext } from 'react';
+import React, { useState, useContext } from 'react';
 import { UserContext } from './../../context/UserContext';
 import Menu from '../../components/Menu/Menu';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Snackbar from '@material-ui/core/Snackbar';
+import Alert from '@material-ui/lab/Alert';
+import { useStyles } from './TeamPage.styles';
+import * as validator from '../../helpers/inputValidator';
+import axios from 'axios';
+
 
 const TeamPage = () => {
   const [user, setUser] = useContext(UserContext);
-  console.log(user);
+  const [invitationDialog, setInvitationDialog] = useState(false);
+  const [email, setEmail] = useState({ input: null, error: false, helperText: null });
+  const [alert, setAlert] = useState({ open: false, message: null, severity: null });
+
+  const classes = useStyles();
+
+  const handleInviationDialogOpen = () => {
+    setInvitationDialog(true);
+  };
+
+  const handleInviationDialogClose = () => {
+    setInvitationDialog(false);
+  };
+
+  const sendInvitation = () => {
+
+    const errors = [];
+    errors.push(validator.validateField(email.input, setEmail, validator.validateEmail));
+    if (!errors.find(error => error === true)) {
+      axios
+        .post(`${process.env.REACT_APP_SERVER_URL}/api/user/invite?email=${email.input.value}`)
+        .then(res => {
+          handleInviationDialogClose();
+          setAlert({ open: true, message: 'An invitation email for the employee was sent successfully!', severity: 'success' });
+        })
+        .catch(err => {
+          handleInviationDialogClose();
+          console.log(err);
+          setAlert({ open: true, message: err.message, severity: 'error' });
+        });
+    }
+  }
+
   return (
-    <Menu>
-      Team page
-    </Menu>
+    <div>
+      <Snackbar open={alert.open} autoHideDuration={600000} onClose={() => setAlert({ open: false, message: null })} anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}>
+        <Alert onClose={() => setAlert({ open: false, message: null })} severity={alert.severity}>
+          {alert.message}
+        </Alert>
+      </Snackbar>
+      <Menu>
+        <Button variant="contained" color="primary" raised="true" onClick={handleInviationDialogOpen}>Invite</Button>
+      </Menu>
+      <Dialog open={invitationDialog} onClose={handleInviationDialogClose} aria-labelledby="form-dialog-title">
```

Comment:
I recommend to move this dialog to separate component, and just to add here <Dialog/> with some props, because otherwise this page file will be huge